components/list/list-item-mixin.js

```diff
@@ -328,18 +323,13 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin-inline-end: -12px;
 			}
 
-			:host([_has-color-slot]) .d2l-list-item-content-extend-separators [slot="outside-control-container"],
-			:host([dir="rtl"][_has-color-slot]) .d2l-list-item-content-extend-separators [slot="outside-control-container"] {
```

Comment:
Was this just here unnecessarily?

Reply:
Yeah, that what I figured, or our more recent changes to remove flagged CSS simplified things to this extent.